### PR TITLE
General housekeeping

### PR DESCRIPTION
**Pull request by nstogner**
* Update readme
* Change test names / directories
* Update development.md guide

NOTE: No actual changes to testing code were made. Changes to workflow files are mostly for readability of files and also of the checks as displayed on PRs.

### CODE REVIEW

File: .github/workflows/build-push.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Create and publish a Docker image
+name: Build and Push
```
Comment (samos123) [inline .github/workflows/build-push.yml:1]: Build and Push Container Image?

File: .github/workflows/build-push.yml [hunk 2]
```diff
@@ -8,44 +8,59 @@ on:
...
 env:
   REGISTRY: ghcr.io
   IMAGE_NAME: substratusai/lingo
 
-# There is a single job in this workflow. It's configured to run on the latest available version of Ubuntu.
+# There is a single job in this workflow.
+# It's configured to run on the latest available version of Ubuntu.
 jobs:
-  build-and-push-image:
+  container:
```
Comment (samos123) [inline .github/workflows/build-push.yml:24]: container as job name seems less descriptive than `build-and-push-image`?

File: .github/workflows/load-tests.yml [hunk 3]
```diff
@@ -4,34 +4,43 @@ run-name: Load tests by @${{ github.actor }}
 on: workflow_dispatch
 
 jobs:
-  perf-tests:
+  k6:
```
Comment (samos123) [inline .github/workflows/load-tests.yml:7]: I probably prefer perf-tests or load-tests. I am not familiar with k6 but if it says load-test or perf-test I would have a general idea of what's supposed to happen in the job.

File: README.md [hunk 4]
```diff
@@ -1,36 +1,41 @@
-# Lingo - K8s LLM Proxy + Scaler
+# Lingo - The lightweight model proxy
```
Comment (samos123) [inline README.md:1]: I do think we should add K8s in the title so it's clear that this only works on K8s and is very focused towards K8s.

What about one of these options?
- The lightweight ML Model proxy and autoscaler for K8s
- Lightweight K8s ML Proxy and Autoscaler
- K8s ML Proxy and Autoscaler

**Review by samos123 — CHANGES_REQUESTED** [review]
Mostly some opinions around naming.

**Review by alpe — COMMENTED** [review]
Good work

File: .github/workflows/build-push.yml (hunk 1 above)
Comment (nstogner) [reply .github/workflows/build-push.yml:1]: Screenshot 2024-01-15 at 8 31 22 PM

Comment (nstogner) [reply .github/workflows/build-push.yml:1]: I am fine reverting, but the reason I made these changes: "Build and Push" as the title of the workflow and "container" as what is being built and pushed. I think it displays nicely on the output.

File: .github/workflows/load-tests.yml (hunk 3 above)
Comment (nstogner) [reply .github/workflows/load-tests.yml:7]: See other response

File: .github/workflows/build-push.yml (hunk 2 above)
Comment (nstogner) [reply .github/workflows/build-push.yml:24]: See other response

File: README.md (hunk 4 above)
Comment (nstogner) [reply README.md:1]: What about leaving the title `# Lingo` and keeping the description in the description section of the repo? I feel like these are too long for an H1.

File: .github/workflows/tests.yml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: ">=1.21.0"
+
+      - name: Run unit tests
+        run: make test-unit
```
[the remark this answers is not in the document]
Comment (nstogner) [reply .github/workflows/tests.yml:24]: Ahh good to know, thanks!

**Review by samos123 — APPROVED** [review]
Names look good based on screenshot

**Pull request merged**